### PR TITLE
Fix pipelines user_agent, ignore CI requests

### PR DESCRIPTION
**Pull request by anton-l**
* Makes sure that the pipeline class takes priority over `_class_name` (so that we count `StableDiffusionImg2Img` too)
* Adds UA logging to the Flax pipeline
* Adds a flag for CI requests to filter them out
* Formats the UA string similarly to `transformers` to fix a weird additional `none/None` inserted into the request

### CODE REVIEW

File: src/diffusers/hub_utils.py [hunk 1]
```diff
@@ -33,6 +46,32 @@
...
 MODEL_CARD_TEMPLATE_PATH = Path(__file__).parent / "utils" / "model_card_template.md"
+SESSION_ID = uuid4().hex
+DISABLE_TELEMETRY = os.getenv("DISABLE_TELEMETRY", False).upper() in ENV_VARS_TRUE_VALUES
+
+
+def http_user_agent(user_agent: Union[Dict, str, None] = None) -> str:
+    """
+    Formats a user-agent string with basic info about a request.
+    """
+    ua = f"diffusers/{__version__}; python/{sys.version.split()[0]}; session_id/{SESSION_ID}"
```
Comment (anton-l) [inline src/diffusers/hub_utils.py:57]: This is how it's done in `transformers`, now here too :)

File: src/diffusers/pipeline_flax_utils.py [hunk 2]
```diff
@@ -301,6 +302,13 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
             allow_patterns = [os.path.join(k, "*") for k in folder_names]
             allow_patterns += [FLAX_WEIGHTS_NAME, SCHEDULER_CONFIG_NAME, CONFIG_NAME, cls.config_name]
 
+            if cls != FlaxDiffusionPipeline:
```
Comment (anton-l) [inline src/diffusers/pipeline_flax_utils.py:305]: Flax pipelines now count too

Comment (patrickvonplaten) [reply src/diffusers/pipeline_flax_utils.py:305]: Cool!

**Review by patrickvonplaten — APPROVED** [review]
Looks good to me!

**Pull request merged**